clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go

```diff
@@ -303,7 +303,11 @@ func (p *podStartupLatencyMeasurement) checkPod(obj interface{}) {
 
 func (p *podStartupLatencyMeasurement) printLatencies(latencies []podLatencyData, header string) {
 	metrics := extractLatencyMetrics(latencies)
-	klog.Infof("%s: 10%% %s: %v", p, header, latencies[(len(latencies)*9)/10:])
+	index := len(latencies) - 100
+	if index < 0 {
+		index = 0
+	}
+	klog.Infof("%s: 100 %s: %v", p, header, latencies[index:])
```

Comment:
Not necessary 100.

s/100/len(latencies) - index/

Reply:
Done.